modules/deploy/partials/cloud/controlplane-api.adoc

```diff
@@ -0,0 +1,288 @@
+:tag-clusters: api:ROOT:cloud-api.adoc#tag--ClusterService
+:tag-networks: api:ROOT:cloud-api.adoc#tag--NetworkService
+:tag-operations: api:ROOT:cloud-api.adoc#tag--OperationService
+:tag-resource-groups: api:ROOT:cloud-api.adoc#tag--ResourceGroupService
+:tag-serverless-regions: api:ROOT:cloud-api.adoc#tag--ServerlessRegionService
+:tag-serverless-clusters: api:ROOT:cloud-api.adoc#tag--ServerlessClusterService
```

Comment:
Should ServerlessRegionService and ServerlessClusterService appear only (and linkable) on the Serverless page?

Reply:
@vladoschreiner should we hide links to the serverless API reference doc endpoints _unless_ the reader is on the Serverless API doc? cc @towfiqa

Reply:
@kbatuigas these links should work, right? (at least on the Serverless tab)

Reply:
@micheleRP yes, once we get the endpoints published to the reference doc, they should work!